docs/cli.rst

```diff
@@ -48,6 +51,12 @@ Or with a filename::
     export FLASK_APP=/path/to/hello.py
     flask run
 
+If you use Flask 0.13 or later you can also set the environment variables
+in a ``.env`` file.  For instance like this::
+
+    FLASK_APP=/path/to/hello.py
+    FLASK_DEBUG=1
+
```

Comment:
@ThiefMaster Ah, okay, I think I understand now. What I don't understand is why in your PR you are using this technique just for additional commands. Why can't this be an alternative to feed the app to the main Flask cli?